src/components/TextField/TextField.tsx

```diff
@@ -37,11 +40,26 @@ class TextField extends React.Component<TextFieldProps & DebouncedOnChangePrivat
         suffix={this.props.suffix}
         underlined={this.props.underlined}
         onChanged={this.props.unifiedOnChange}
+        componentRef={this.getRef}
         {...getBaseTextFieldProps(this.props)}
       />
     );
   }
 
+  public focus = () => {
+    const el = this.fabricTextFieldElement;
+    if (el) {
+      const valueLength = this.props.value ? this.props.value.length : 0;
+
+      el.focus();
+      el.setSelectionRange(valueLength, valueLength);
+    }
+  };
+
+  private getRef = (ref: ITextField | null) => {
```

Comment:
this should probably be named `setRef` since it's a setter and not returning anything

Reply:
good eye